inc/util/any.h

```diff
@@ -357,6 +367,11 @@ struct bad_any_cast: std::bad_cast {
     }
 };
 
+// should we make these more generic? otherwise the user has
+// to define a new set for every distinct Policy
+//   template<class ValueType, class Policy>
+//   ValueType any_cast(const AnyContainer<Policy> &operand) {
+//       ...
```

Comment:
I dislike `any_cast` as specified in the standard.  I provide this API just for compliance.  The instance function `AnyContainer<Policy>::state` is at the moment my proposal to fix issues of `any_cast`